app/models/account.rb

```diff
@@ -562,10 +562,6 @@ def provider_id_for_audits
     end
   end
 
-  def independent_mapping_rules_enabled?
-    provider_can_use?(:api_as_product) || provider_can_use?(:independent_mapping_rules)
```

Comment:
I guess that we can also get rid of these 2 lines:
https://github.com/3scale/porta/blob/99901577f26e32c57a086f7c3c2b0e0d9240ca96/config/examples/rolling_updates.yml#L19-L20

Reply:
#2274